adafruit_bme680.py

```diff
@@ -457,7 +471,7 @@ def _read(self, register, length):
                 print("\t$%02X => %s" % (register, [hex(i) for i in result]))
             return result
 
-    def _write(self, register, values):
+    def _write(self, register: int, values: int) -> None:
```

Comment:
The type for values should be something that implements the buffer protocol and can be read from, so its that ReadableBuffer:

```suggestion
    def _write(self, register: int, values: ReadableBuffer) -> None:
```